docker/jupyter/CERN_Root_Certification_Authority_2.pem

```diff
@@ -0,0 +1,38 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
can we reuse the same one as hep?

Reply:
it is difficult cause it's not in docker context and either I have to copy this or I have to copy files from `docker/jupyter` to `inspirehep/backend`